docs/user-guide/install/edge/windows.md

```diff
@@ -31,7 +31,7 @@ Instructions below are provided for Windows 10/8.1/8/7 32-bit/64-bit.
 Download and extract the package.
 
 ```bash
-https://dist.thingsboard.io/tb-edge-windows-3.3.zip
+https://dist.thingsboard.io/tb-edge-windows-3.3.3.zip
```

Comment:
Wat is the point of editing versions in _config.yml  if you don't use them here? Please introduce new variable in the _config.yml if you need it but let's remove all the hard-coded links to download URLs.

Reply:
Fixed. Using site.release.edge_ver in installation instructions. Hardcoded 3.3.3 version variable remains only in upgrade instructions.